frontend/src/pages/LandingPage.js

```diff
@@ -49,10 +49,12 @@ class LandingPage extends Component {
                 <PageHeader />
                 <div>
```

Comment:
I feel like there should be some extra space between the moments list and the logo, at least for how the landing page is organized currently. This can be done by adding `margin: 3em 0` to the CSS for this div.

Reply:
Yes, I mentioned this to Justin as well. I am planning on putting a "create moment" button underneath the caption and adjust the spacing accordingly after that addition.

Reply:
If you are going to add a create moment button, I would place it either right/left aligned above the moment list. I think it would look there.